torch/_torch_docs.py

```diff
@@ -1500,6 +1500,58 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov,
+           r"""
+cov(input1, input2=None, rowvar=True, bias=None, ddof=None, fweights=None, aweights=None) -> Tensor
+
+Estimates a covariance matrix, given input tensor data and weights (optional).
+
+Args:
+    input1 (Tensor): A 1-D or 2-D tensor containing multiple variables and observations. 
+        Each row of :attr:`input1` represents a variable, and each column a 
+        single observation of all those variables. Also see :attr:`rowvar` below.
+    input2 (Tensor, optional): An additional set of variables and observations. 
```

Comment:
I'm confused by this in the NumPy documentation because this tensor isn't related to the computation. How is this "additional set" used? How does it change the output?

Reply:
Given, that `input` is of the form `m*n` matrix, `other` would also be of the form `g*n` matrix.

How it changes things is that other is basically appended to input along the first dimension. Since each row defines variables and each column is observation, what this does is it increases the number of variable for each observation.

Therefore instead of an `m*m` output we will get a `(m+g)*(m+g)` matrix. Hope that makes sense? @mruberry

Reply:
That makes sense. Let's be sure to clarify that in the docs.

I wonder why it was designed that way? Seems odd to not just require the user stack the matrices.